narwhals/_dask/expr.py

```diff
@@ -919,116 +919,6 @@ def is_finite(self: Self) -> Self:
             returns_scalar=self._returns_scalar,
         )
 
-    def rolling_sum(
```

Comment:
these rolling functions are marked as unstable

they're currently xfailed for dask anyway

i'd suggest we just remove these for now until https://github.com/narwhals-dev/narwhals/discussions/1657 is resolved